### DISCUSSION TITLE
Permission Module

### DISCUSSION THREAD

**Discussion by mabdalla123** [abpframework/abp#19099] [Q&A]
in abp how can i implement a rule that a user cant have multiple permissions in a single permission group other than admin  and how to remove select all in permission group

**Comment by maliming** [comment]
hi

You can check and custom the source code of [permission-management](https://github.com/abpframework/abp/tree/dev/modules/permission-management) module

**Comment by mabdalla123** [comment]
![image](https://github.com/abpframework/abp/assets/41789474/67143d06-cf33-4880-8f8f-5d445972bd16)
i have created this class and it's working for me ,
